src/components/Navbar.tsx

```diff
@@ -0,0 +1,69 @@
+import { AiOutlineClose, AiOutlineMenu } from "react-icons/ai";
+import { useState } from "react";
+
+const Navbar = () => {
+  const [showMenu, setShowMenu] = useState(false);
+
+  const handleToggleMenu = () => {
+    setShowMenu((showMenu) => !showMenu);
+  };
+
+  return (
+    <nav className="py-8">
+      <div className="container flex justify-between items-center">
+        <div className="text-white uppercase text-2xl font-bold">Catalyst</div>
+        <div className="hidden md:block">
+          <ul className="flex items-center gap-4 text-ternary text-lg">
+            <li>
+              <a href="">Home</a>
+            </li>
+            <li>
+              <a href="">About</a>
+            </li>
+            <li>
+              <a href="">Features</a>
+            </li>
+            <li>
+              <a href="">Contact</a>
+            </li>
```

Comment:
can you update the links to navigate to the desired section, for example `#about` to navigate to the about section.